packages/vue-components/src/panels/PanelBase.js

```diff
@@ -141,8 +141,9 @@ export default {
         requestAnimationFrame(() => {
           // To enable behaviour of auto window scrolling during panel collapse
           if (this.$el.getBoundingClientRect().top < 0) {
+            const headerHeight = jQuery('header[fixed]').height() || 0;
```

Comment:
a really minor nit for keeping everything under the panel implementation -- would this work?
```
const el = $refs.anchor || this.$el;
el .getBoundingClientRect......
```

Reply:
Hmm added `ref="anchor"` to the `span`s and doesn't seem to be working... The panel header gets hidden under the navbar still.

Reply:
try adding `display: inline-block` to the scoped anchor class here

Reply:
Slightly better but half of it is still under the navbar... When lower navbar is present, it gets hidden again 😅  

![image](https://user-images.githubusercontent.com/47470981/124943854-6c54f280-e03f-11eb-9cdb-cf3d2080352f.png)

Reply:
Looks like the sibling after span.anchor is interfering 🤣...  let's leave it for now and revisit later